LayoutTests/media/media-hevc-video-as-img.html

```diff
@@ -0,0 +1,9 @@
+<html>
+<head>
+    <meta name="fuzzy" content="maxDifference=0-84; totalPixels=0-293510" />
```

Comment:
That's a lot of fuzziness. Why so many pixels?

Reply:
> That's a lot of fuzziness. Why so many pixels?

Each decoder on each platform produce a different output with hevc content. The png was produced using ffmpeg software decoder. 
The aim of the test really is to make sure we have an image out. So fuzziness doesn't really matter too much so long thst the difference is overall small